fbw-common/src/systems/instruments/src/EFB/Ground/Pages/Fuel/A320_251N/A320Fuel.tsx

```diff
@@ -66,6 +72,12 @@ const TankReadoutWidget = ({
   );
 };
 
+const RefuelRate = {
+  REAL: '0',
+  FAST: '1',
+  INSTANT: '2',
+};
```

Comment:
Why not enum?

Reply:
Reason: Poor Execution on my Side 😅
Copy and pasted from the Refuel JavaScript File, in Case I should re-enum more existing Code ;)